api/source/github/events/handlers/event.ts

```diff
@@ -42,14 +43,22 @@ export const runEventRun = async (
     settings: settings.installationSettings,
   }
 
+  // Manually set the Octokit API object so that danger.github.api works
+  const dangerDSL = {
+    github: {
+      ...event,
+      api: await octokitAPIForPeril(settings.installationID, token),
+    },
+  }
+
```

Comment:
But I'm totally cool with this, it's a good idea IMO

If you replace the GitHub API in the snapshot with some dumb object (yeah, it kinda undermines the test but I'd rather that then a test which mysteriously breaks later)